### ISSUE TITLE
The performance of the pot_result in MSL was only 0.77.

### ISSUE THREAD

**Issue by DingOUCer** [ML4ITS/mtad-gat-pytorch#46]
Hello, when I trained the msl dataset using the default parameters, the results showed that using the POT method had a significantly different effect from the others, and it also didn't match the data in the article. Could you please explain why this is the case? Below are my parameters and the training result log:

 { "dataset": "MSL",
"group": "1-1",
"lookback": 100,
"normalize": true,
"spec_res": false,
"kernel_size": 7,
"use_gatv2": true,
"feat_gat_embed_dim": null,
"time_gat_embed_dim": null,
"gru_n_layers": 1,
"gru_hid_dim": 150,
"fc_n_layers": 3,
"fc_hid_dim": 150,
"recon_n_layers": 1,
"recon_hid_dim": 150,
"alpha": 0.2, "epochs": 10,
"val_split": 0.1, "bs": 256,
"init_lr": 0.001, "shuffle_dataset": true,
"dropout": 0.3, "use_cuda": true,
"print_every": 1,
"log_tensorboard": true,
"scale_scores": false,
"use_mov_av": false,
"gamma": 1,
"level": null,
"q": null,
"dynamic_pot": false,
"comment": ""
}
###########################################################################

{
"epsilon_result": {
"f1": 0.9549911857376504,
"precision": 0.943007781894291,
"recall": 0.9672933286540133,
"TP": 7512.0,
"TN": 65409.0,
"FP": 454.0,
"FN": 254.0,
"threshold": 0.5882286354899406,
"latency": 38.99988181853994,
"reg_level": 0.0 },
"pot_result": {
"f1": 0.7737982153144096,
"precision": 0.9935392671238861,
"recall": 0.6336595407756123,
"TP": 4921.0,
"TN": 65831.0,
"FP": 32.0,
"FN": 2845.0,
"threshold": 0.913154621374092,
"latency": 63.8633460756996 },
"bf_result": {
"f1": 0.9595657981226434,
"precision": 0.951970598210657,
"recall": 0.9672933286540133,
"TP": 7512.0,
"TN": 65484.0,
"FP": 379.0,
"FN": 254.0,
"threshold": 0.6269000000000006,
"latency": 41.2422992657598 }
}
